src/handlers.js

```diff
@@ -10,6 +10,33 @@ const {consume} = require('taskcluster-lib-pulse');
 const debugPrefix = 'taskcluster-github:handlers';
 const debug = Debug(debugPrefix);
 
+const TITLES = { // maps github checkruns statuses and conclusions to titles to be displayed
+  success: 'Success',
+  failure: 'Failure',
+  neutral: 'It is neither good nor bad',
+  cancelled: 'Cancelled',
+  timed_out: 'Timed out',
+  action_required: 'Action required',
+  queued: 'Queued',
+  in_progress: 'In progress',
+  completed: 'Completed',
+};
+
+const CONCLUSIONS = { // maps queue exchange status to github checkrun conclusion
+  completed: 'success', // TODO: remove this stupid rule from es-lint
+  failed: 'failure',
+  exception: 'failure',
+  'deadline-exceeded': 'timed_out',
+  canceled: 'cancelled',
+  superseded: 'neutral', // is not relevant anymore
+  'claim-expired': 'neutral', // will be retried
```

Comment:
I _think_ that you will only get a pulse message once the final retry has an exception with `claim-expired`. In that case it would be a failure here.

Reply:
I thought that would mean a retry - i.e. a new task is created? (I don't know how queue is working here, tho. So I'll trust you, I think.... So, is that true for the rest of `neutral`s here? Should I just replace that with `failure`?)

Reply:
It's a bit confusing -- `retry` vs `rerun`. A rerun happens within a task and a retry creates a new task. Exceptions result in reruns. I might've even said it results in a retry at some point when I was being imprecise so the confusion is easy to create.

Yeah, let's call it a failure.